app/lib/widgets/move_card.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:manvsim/models/location.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import '../services/location_service.dart';
+
+class MoveCard extends StatefulWidget {
+  final List<Location> locations;
+  final Function(Location location) onPerform;
+
+  const MoveCard({super.key, required this.locations, required this.onPerform});
+
+  @override
+  State<MoveCard> createState() => _MoveCardState();
+}
+
+class _MoveCardState extends State<MoveCard> {
+  late Future<List<Location>?> futureLocationIdList;
+
+  int? selectedLocationIndex;
+
+  @override
+  void initState() {
+    futureLocationIdList = LocationService.fetchLocations();
+    super.initState();
+  }
+
+  bool isSelected(int index) {
+    return selectedLocationIndex != null && selectedLocationIndex == index;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+        child: ExpansionTile(
+      title: Text(AppLocalizations.of(context)!.patientMoveSelection),
+      controlAffinity: ListTileControlAffinity.leading,
+      // removes border on top and bottom
+      shape: const Border(),
+      childrenPadding: const EdgeInsets.only(left: 8.0),
+      children: [
+        ListView.builder(
+          shrinkWrap: true, // nested scrolling
+          physics: const ClampingScrollPhysics(),
+          itemCount: widget.locations.length,
+          itemBuilder: (context, index) {
+            Location location = widget.locations[index];
+            return Card(
+                color: isSelected(index) ? Colors.lightGreen : Colors.grey,
+                child: ListTile(
+                    title: Text(location.name),
+                    onTap: () {
+                      setState(() {
+                        selectedLocationIndex =
+                            selectedLocationIndex == index ? null : index;
+                      });
+                    }));
+          },
+        ),
+
+        if (selectedLocationIndex != null)
+          Column(children: [
```

Comment:
Is it desired here to have a new Column, or should we use the spread operator to integrate into `children` directly?